### DISCUSSION TITLE
Packaged ARM64 natives missing in nuget?

### DISCUSSION THREAD

**Discussion by thoron** [microsoft/onnxruntime#10624] [General] [answered]
https://github.com/microsoft/onnxruntime/blob/d6a8cba27302bcd494040f32511568f81d70bace/tools/nuget/generate_nuspec_for_native_nuget.py#L69

I believe `arm64` `cpu_arch` should be included when packaging the Linux natives for nuget. Currently only aarch64 is package and this is not a valid RID (if you publish with `dotnet publish`)

**Comment by snnn** [comment, marked as answer]
Good catch! Would you like to submit a PR? Add a line below to replace 'aarch64' to arm64.

**Reply by thoron** [reply]
Yes, I will create the PR 👍

**Reply by thoron** [reply]
PR created: https://github.com/microsoft/onnxruntime/pull/10637

**Comment by snnn** [comment]
Later on I will publish a new package with your fix to: https://aiinfra.visualstudio.com/PublicPackages/_packaging?_a=package&feed=ORT-Nightly&package=Microsoft.ML.OnnxRuntime&protocolType=NuGet&version=1.11.0-dev-20220223-0319-e0d1d6906
